### PR TITLE
Add tests for PersistentNetNames and PersistentNetNamesInitramfs

### PR DESCRIPTION
**Pull request by drehak**
Both actors are almost identical except for the name and phase.

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/persistentnetnames/tests/test_persistentnetnames.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from leapp.libraries.common import persistentnetnames
+from leapp.models import Interface, PCIAddress, PersistentNetNamesFacts
+from leapp.snactor.fixture import current_actor_context
+
+
+def interface_mocked(i=0):
+    return Interface(
+        name='n'+str(i),
```
Comment (ZhukovGreen) [inline repos/system_upgrade/el7toel8/actors/persistentnetnames/tests/test_persistentnetnames.py:10]: Just out of curiosity compared with the way I would do that and this is the result:
```python

In [2]: %timeit "n{}".format(1)                                                                                         
156 ns ± 0.644 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [3]: %timeit "n" + str(1)                                                                                            
168 ns ± 0.253 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```
** this is not th requirement to change **

Comment (drehak) [reply repos/system_upgrade/el7toel8/actors/persistentnetnames/tests/test_persistentnetnames.py:10]: No, this is a good point. Thanks!

**Comment by drehak** [comment]
Am I doing this right?
```
---------- coverage: platform linux2, python 2.7.18-final-0 ----------
Name                                                                           Stmts   Miss  Cover
--------------------------------------------------------------------------------------------------
actors/persistentnetnames/actor.py                                                11      0   100%
actors/persistentnetnames/tests/test_persistentnetnames.py                        16      0   100%
actors/persistentnetnamesinitramfs/actor.py                                       11      0   100%
actors/persistentnetnamesinitramfs/tests/test_persistentnetnamesinitramfs.py      16      0   100%
--------------------------------------------------------------------------------------------------
TOTAL                                                                             54      0   100%
```

**Comment by drehak** [comment]
@ZhukovGreen Thanks! :)

**Pull request merged**